element-connector/src/main/java/org/eclipse/californium/elements/UdpMulticastConnector.java

```diff
@@ -27,6 +35,7 @@
  * Protocol (CoAP) to the registered multicast group.
  */
 public class UdpMulticastConnector extends UDPConnector {
+	public static final Logger LOGGER = LoggerFactory.getLogger(UdpMulticastConnector.class.getName());
```

Comment:
why not ? LoggerFactory.getLogger(UdpMulticastConnector.class);

Reply:
Just copied the UDPConnector stuff.